Makefile.rhpam

```diff
@@ -19,20 +19,20 @@ BUNDLE_IMGS ?= $(BUNDLE_IMG)
 CEKIT_CMD := cekit -v --redhat ${cekit_option}
 
 all: generate manifests container-build
-	echo "calling RHPAM all......................................"
+	echo "calling IBM BAMOE all......................................"
```

Comment:
Not sure the scripts need update. we just have to update what is customer view.

but the echoes should be harmless :)

Reply:
yes :D